code/modules/atmospherics/machinery/bluespace_vendor.dm

```diff
@@ -112,6 +112,20 @@
 	if(!selected_gas)
 		return
 	var/gas_path = gas_id2path(selected_gas)
+
+	var/list/gases = connected_machine.bluespace_network.gases
+	var/gas_found = FALSE
+	for(var/gas_id in gases)
```

Comment:
Why are you doing a for loop here, you already have the path you want to check don't you?

Reply:
So I just do `if(!connected_machine.bluespace_network.gases[gas_path])`?

Reply:
I don't see why not

Reply:
done